packages/web-react/src/components/Modal/demo/index.tsx

```diff
@@ -1,28 +1,28 @@
+// Because there is no `dist` directory during the CI run
+/* eslint-disable import/no-extraneous-dependencies, import/extensions, import/no-unresolved */
 import React from 'react';
 import ReactDOM from 'react-dom/client';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore: No declaration file
+import icons from '@lmc-eu/spirit-icons/dist/icons';
 import DocsSection from '../../../../docs/DocsSections';
-import Modal from './Modal';
-import ModalWithCustomHeight from './ModalWithCustomHeight';
-import ModalWithLongText from './ModalWithLongText';
-import ModalWithCheckbox from './ModalWithCheckbox';
-import ModalStacked from './ModalStacked';
+import { IconsProvider } from '../../../context';
+import ModalDefault from './ModalDefault';
+import ModalScrollingLongContent from './ModalScrollingLongContent';
+import ModalStacking from './ModalStacking';
 
 ReactDOM.createRoot(document.getElementById('root') as HTMLElement).render(
   <React.StrictMode>
-    <DocsSection title="Modal">
-      <Modal />
-    </DocsSection>
-    <DocsSection title="Modal with Custom Height">
-      <ModalWithCustomHeight />
-    </DocsSection>
-    <DocsSection title="Modal with Long Text">
-      <ModalWithLongText />
-    </DocsSection>
-    <DocsSection title="Modal with Checkbox">
-      <ModalWithCheckbox />
-    </DocsSection>
-    <DocsSection title="Modal Stacked">
-      <ModalStacked />
-    </DocsSection>
+    <IconsProvider value={icons}>
```

Comment:
somewhere (eg. Alert) we wrap each item inside demos and here we wrap the whole page. I feel like this is better, or maybe it could be a part of some wrapper (Docs section?). We don't need to do anything now, we just might look into it in the future. cc @literat